### PR TITLE
NIFI-5677 Added note to clarify why modifying/creating variables not …

### PR DESCRIPTION
**Pull request by andrewmlim**
…local changes in versioned flows

### CODE REVIEW

File: nifi-docs/src/main/asciidoc/user-guide.adoc [hunk 1]
```diff
@@ -1928,7 +1928,9 @@ The following actions are not considered local changes:
 * modifying sensitive property values
 * modifying remote process group URLs
 * updating a processor that was referencing a non-existent controller service to reference an externally available controller service
-* modifying variables
+* creating or modifying variables
```
Comment (ijokarumawak) [inline nifi-docs/src/main/asciidoc/user-guide.adoc]: A bit nit-picky, but how about deleting variables?

Comment (andrewmlim) [reply nifi-docs/src/main/asciidoc/user-guide.adoc]: That's a good point.  Will add that scenario.

Comment (andrewmlim) [reply nifi-docs/src/main/asciidoc/user-guide.adoc]: Added "deleting variables" scenario.  PR read for review/merge if no other issues raised. Thanks @ijokarumawak !

**Comment by ijokarumawak** [comment]
LGTM, +1. Merging. Thanks @andrewmlim!

**Pull request closed without merging**